src/commandSocket.cpp

```diff
@@ -93,20 +93,29 @@ void CommandSocket::connect(const std::string& ip, int port)
 {
     private_data->socket = new Arcus::Socket();
     //private_data->socket->registerMessageType(1, &Cura::ObjectList::default_instance());
-    private_data->socket->registerMessageType(1, &cura::proto::Slice::default_instance());
-    private_data->socket->registerMessageType(2, &cura::proto::SlicedObjectList::default_instance());
-    private_data->socket->registerMessageType(3, &cura::proto::Progress::default_instance());
-    private_data->socket->registerMessageType(4, &cura::proto::GCodeLayer::default_instance());
-    private_data->socket->registerMessageType(5, &cura::proto::ObjectPrintTime::default_instance());
-    private_data->socket->registerMessageType(6, &cura::proto::SettingList::default_instance());
-    private_data->socket->registerMessageType(7, &cura::proto::GCodePrefix::default_instance());
+    private_data->socket->registerMessageType(&cura::proto::Slice::default_instance());
+    private_data->socket->registerMessageType(&cura::proto::SlicedObjectList::default_instance());
+    private_data->socket->registerMessageType(&cura::proto::Progress::default_instance());
+    private_data->socket->registerMessageType(&cura::proto::GCodeLayer::default_instance());
+    private_data->socket->registerMessageType(&cura::proto::ObjectPrintTime::default_instance());
+    private_data->socket->registerMessageType(&cura::proto::SettingList::default_instance());
+    private_data->socket->registerMessageType(&cura::proto::GCodePrefix::default_instance());
 
     private_data->socket->connect(ip, port);
+
+    log("Connecting to %s:%i", ip.c_str(), port);
```

Comment:
Having the Connecting and Connected messages separately helps with debugging, so that you can properly distinguish whether it just fails to connect or something else is going on.